unit_tests/test_latte_thread.py

```diff
@@ -225,3 +226,21 @@ def test_get_latte_operation_type(cmd, expected_operation_type):
     for fn_param in fn_params:
         result = get_latte_operation_type(cmd % fn_param)
         assert expected_operation_type == result
+
+
+@pytest.mark.parametrize(
+    "cmd,items", (
+        ("%s --tag=latte-prepare-01 -q -r 500", ["latte-prepare-01"]),
+        ("%s --tag latte-main-01 -q -r 500", ["latte-main-01"]),
+        ("%s --tag  latte-prepare-01,write  -q -r 500", ["latte-prepare-01", "write"]),
+        ("%s --tag=latte-main-01,read    -q -r 500", ["latte-main-01", "read"]),
+        ("%s  --tag=latte-main-01  --tag   write,table1    -q -r 500", ["latte-main-01", "write", "table1"]),
+        ("%s --tag=latte-main-01,read  --tag table2  -q -r 500", ["latte-main-01", "read", "table2"]),
```

Comment:
can also add case for `--tag` being at the end of command

Reply:
Done